src/js/utils/utils.js

```diff
@@ -61,4 +61,8 @@ export function humanFileSize(bytes) {
         ++u;
     } while(Math.abs(bytes) >= thresh && u < units.length - 1);
     return bytes.toFixed(1)+' '+units[u];
+}
+
+export function openWindow720x720(link) {
+    window.open(link, 'wordpress', 'toolbar=0,status=0,width=720,height=700');
```

Comment:
`wordpress` should be `_blank`. Don't want the a href target to be WordPress specific.